x-pack/test/detection_engine_api_integration/basic/tests/coverage_overview.ts

```diff
@@ -341,6 +341,51 @@ export default ({ getService }: FtrProviderContext): void => {
             },
           });
         });
+
+        it('returns response filtered by enabled and disabled rules equal to response if enabled and disabled are not set', async () => {
+          const expectedRule1 = await createRule(supertest, log, {
+            ...getSimpleRule('rule-1'),
```

Comment:
`getSimpleRule('rule-1', false)` would be a more robust expression, in case someone removes the `enabled` parameter or changes its default value. This can be important for this particular test that actually depends on this.